tests/test_utils.py

```diff
@@ -190,3 +190,46 @@ def test_convert_units():
     truth = r_kpc * (1.0 / d_a) * (180. / np.pi) * 60.
     assert_allclose(utils.convert_units(r_kpc, 'kpc', 'arcmin', redshift, cosmo),
                     truth, **TOLERANCE)
+
+    
+    
+def test_shape_conversion():
+    """ Test the helper function that convert user defined shapes into
+    epsilon ellipticities or reduced shear. Both can be used for the galcat in 
+    the GalaxyCluster object"""
+    
+    def ellipticities(q20,q11,q02):
```

Comment:
So, I think this def ellipticities function should either live in utils.py or elsewhere because this could be generally useful.

Reply:
also, can this be named, 
`def build_ellipticities(q20, q11, q02):
`